modules/gdscript/gdscript_parser.cpp

```diff
@@ -3481,22 +3481,22 @@ bool GDScriptParser::network_annotations(const AnnotationNode *p_annotation, Nod
 		}
 		for (int i = last; i >= 0; i--) {
 			String mode = p_annotation->resolved_arguments[i].operator String();
-			if (mode == "any") {
-				rpc_config.rpc_mode = Multiplayer::RPC_MODE_ANY;
-			} else if (mode == "auth") {
+			if (mode == "any_peer") {
+				rpc_config.rpc_mode = Multiplayer::RPC_MODE_ANY_PEER;
+			} else if (mode == "authority") {
 				rpc_config.rpc_mode = Multiplayer::RPC_MODE_AUTHORITY;
-			} else if (mode == "sync") {
+			} else if (mode == "call_local") {
 				rpc_config.sync = true;
-			} else if (mode == "nosync") {
+			} else if (mode == "no_call_local") {
```

Comment:
maybe call_remote ? although I thought this was the default.

Reply:
yes, that is the default.